packages/block-editor/src/hooks/block-hooks.js

```diff
@@ -83,12 +78,12 @@ function BlockHooksControlPure( props ) {
 							// Any of the current block's child blocks (with the right block type) qualifies
 							// as a hooked first or last child block, as the block might've been automatically
 							// inserted and then moved around a bit by the user.
-							candidates = getBlock( props.clientId ).innerBlocks;
+							candidates = getBlock( clientId ).innerBlocks;
 							break;
 					}
 
 					const hookedBlock = candidates?.find(
-						( { name } ) => name === block.name
+						( candidate ) => name === candidate.name
```

Comment:
I just noticed that this broke the toggle 😬 

I've filed a fix: https://github.com/WordPress/gutenberg/pull/57956.

Reply:
Whoops, sorry!